ci/lava/tests/boot-component-update.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2019, Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+metadata:
+    format: Lava-Test Test Definition 1.0
+    name: Boot component update
+    description: Boot component update
+
+parameters:
+    #
+    # virtual_env: specifies the Python virtual environment
+    #
+    virtual_env:
+
+    #
+    # component_name: specifies the bootloader component to update
+    #
+    component_name:
+
+    #
+    # payload_url: specifies the full url of the boot component payload to update
+    #
+    payload_url:
+
+    #
+    # update_method: specifies the method used to do the update (mbl-cli or pelion)
+    #
+    update_method:
+
+run:
+    steps:
+        # Activate virtual environment
+        - . $virtual_env/bin/activate
+        - pytest --verbose --debug-output -s ./ci/lava/tests/test-boot-component-update.py --update-component-name $component_name --update-payload-url $payload_url --update-method $update_method
```

Comment:
```suggestion
        - pytest --verbose -s ./ci/lava/tests/test-boot-component-update.py --update-component-name $component_name --update-payload-url $payload_url --update-method $update_method
```

Reply:
I left it on purpose. Until we don't have a stable testing on this on all platforms I would prefer to keep it.